packages/ingest/granule.js

```diff
@@ -381,7 +363,21 @@ class Granule {
    */
   async verifyFile(file, bucket, key, options = {}) {
     const [type, value] = await this.retrieveSuppliedFileChecksumInformation(file);
-    if (!type || !value) return [null, null];
+    if (!type || !value) {
```

Comment:
Correct, because we need both a `type` and `value` to verify a checksum, unless we want to assume that a checksum without a type is a `cksum` checksum. Our checksum functions support algorithms other than `cksum`, and some providers use `md5`. CNM for example assumes that the checksumType is `md5`. I will improve the logged statement.

Reply:
I think this is good. I bring this up because I want to make sure that changing this value doesn't break the assumption that CNM messages have that [cksum type is optional and defaults to cksum](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cloud+Notification+Mechanism#CloudNotificationMechanism-ResponseMessageFields#granule-fields).

Reply:
The CNM default is actually `it is assumed to be md5.`, but their CNM translation lambda should hand populating the `checksumType` field with that default value if it does not exist in the original CNM message.

Reply:
(At least it will if they accept my PR to that function. Yay insider knowledge!)